src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,76 @@
-import React from 'react';
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useContext, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { GlobalContext } from '../AppContext';
 
 export const NewCommentForm: React.FC = () => {
+  const {
+    handleAddComment,
+    postActiveId,
+    setIsCommentSuccess,
+    isCommentSuccess,
+  } = useContext(GlobalContext);
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
```

Comment:
you can create one state with `object: CommentData`, because `CommentData` already contains all needed properties